src/reducers/helpers/version-comparison.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Compares two version strings.
+ * @param {*} a - The first version to compare.
+ * @param {*} b - The second version to compare.
+ */
+function compareVersions(a, b) {
+  let i;
+  let cmp;
+  const aSplit = (`${a}`).split('.');
+  const bSplit = (`${b}`).split('.');
+  const len = Math.max(aSplit.length, bSplit.length);
+  for (i = 0; i < len; i += 1) {
+    if (aSplit[i] === undefined) {
+      aSplit[i] = '0';
+    }
+    if (bSplit[i] === undefined) {
+      bSplit[i] = '0';
+    }
+    cmp = parseInt(aSplit[i], 10) - parseInt(bSplit[i], 10);
+    if (cmp !== 0) {
+      return (cmp < 0 ? -1 : 1);
+    }
+  }
+
+  return 0;
+}
+
+export { compareVersions as default };
```

Comment:
Consider using an external library like https://www.npmjs.com/package/compare-versions (well supported) to cut down on code here + test cases.

(Obviously this is a pretty trivial dependency, so... either way.)

Reply:
+1 to using an existing package for comparing versions

Reply:
My apologies. I had the impression that the team wanted to introduce as few dependencies as possible. That package is licensed under MIT so I'll make the change.